calculator-project/src/CSS/buttons.css

```diff
@@ -0,0 +1,45 @@
+#plus{  
```

Comment:
by adding `#plus, #minus, #times(etc){ background-color: (insert color here)}` all this CSS could have been done in a line or two